pkg/webhook/mutation.go

```diff
@@ -186,6 +186,13 @@ func (h *mutationHandler) mutateRequest(ctx context.Context, req *admission.Requ
 		return admission.Errored(int32(http.StatusInternalServerError), err)
 	}
 
+	// It is possible for namespace and name to not be populated on an object.
+	// This is true when defaulting to the "default" namespace.
```

Comment:
> It is possible for namespace and name to not be populated on an object.

It seems this is also true beyond just "default" namespace.

Reply:
Have you noticed this in other situations?

Reply:
Ah, you commented on the bug: "I was able to repro this issue with 3.7.0 and k8s v1.21.2 in both default namespace and "test" namespace. Also confirming #1760 fixes the issue."

Reply:
updated comment to remove mention of the "default" namespace